poll/poll_request.go

```diff
@@ -32,7 +32,17 @@ func NewPollRequest(s string) (*PollRequest, error) {
 				return nil, fmt.Errorf("Unexpected Error: ChannelID in request is empty.")
 			}
 		case "text":
-			re := regexp.MustCompile("`([^`]+)`(.+)")
+      var re *(regexp.Regexp)
```

Comment:
And I think that it should create new private function for parsing `text` field.